### PR TITLE
Fix example searching

### PR DESCRIPTION
**Pull request by mikkel**
Addresses issues found when running through search for each project.

* [x] distribution search
* [x] static search
* [x] colorizer search
* [x] alignment search
* [x] classification search
* [x] chargan search

### CODE REVIEW

File: hypergan/discriminators/common.py [hunk 1]
```diff
@@ -5,18 +5,16 @@ def repeating_block(component, net, depth, filter=3):
     ops = component.ops
     config = component.config
     layer_regularizer = config.layer_regularizer
-    filter_size_w = filter
-    filter_size_h = filter
-    filter = [1,filter_size_w,filter_size_h,1]
-    stride = [1,filter_size_w,filter_size_h,1]
+    ksize = [1,filter-1,filter-1,1]
+    stride = [1,filter-1,filter-1,1]
```
Comment (martyn) [inline hypergan/discriminators/common.py]: This breaks networks in 0.9

File: hypergan/discriminators/common.py [hunk 2]
```diff
@@ -5,18 +5,16 @@ def repeating_block(component, net, depth, filter=3):
...
     for i in range(config.block_repeat_count-1):
         net = config.activation(net)
         if layer_regularizer is not None:
             net = component.layer_regularizer(net)
         net = ops.conv2d(net, 3, 3, 1, 1, depth)
         print("[discriminator] hidden layer", net)
 
-    net = tf.nn.avg_pool(net, ksize=filter, strides=stride, padding='SAME')
+    net = tf.nn.avg_pool(net, ksize=ksize, strides=stride, padding='SAME')
```
Comment (martyn) [inline hypergan/discriminators/common.py]: this too

File: hypergan/gans/alpha_gan.py [hunk 3]
```diff
@@ -82,13 +82,7 @@ def create(self):
...
 
             # project the output of the autoencoder
-            projection_input = ops.reshape(encoder.sample, [ops.shape(encoder.sample)[0],-1])
-            projections = []
-            for projection in uniform_encoder.config.projections:
-                projection = uniform_encoder.lookup(projection)(uniform_encoder.config, self.gan, projection_input)
-                projection = ops.reshape(projection, ops.shape(encoder.sample))
-                projections.append(projection)
-            z_hat = tf.concat(axis=3, values=projections)
+            z_hat = encoder.sample
```
Comment (martyn) [inline hypergan/gans/alpha_gan.py]: Breaks 0.9 alphagan

**Comment by mikkel** [comment]
This is already in 0.10, closing this pr

**Pull request closed without merging**